docs/development.md

```diff
@@ -233,3 +233,61 @@ This is GitHub user that does all the automation.
 | Script      | What it is doing                          | Access via                                    |
 |-------------|-------------------------------------------|-----------------------------------------------|
 | .travis.yml | Uploading binaries to GitHub release page | Personal access token `deploy-github-release` |
+
+
+## Licenses
+
+A software license is a legal instrument governing the use or redistribution of software.
```

Comment:
No need to describe what a license is, developers know licensing.

Reply:
I sometimes don't quite understand if these docs are intended for beginners or only experienced developers starting to use odo anyway these can be looked up on wiki as well so I'll remove this

Reply:
I think it was ok, as it is better to write it for beginners, experienced developers can always just skip what they already know. 

It is better to write it with assumptions that readers don't know this, as you can never be sure.

But it is up to you guys, go with what you both agree on ;-)